dotnet-dependency-injection/DependencyInjectionLifetimeScopes/DependencyInjectionLifetimeScopes/Startup.cs

```diff
@@ -0,0 +1,54 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+
+namespace DependencyInjectionLifetimeScopes
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+
+            services.AddControllers();
+            services.AddTransient<IMyService, MyService>();
```

Comment:
Readers should have different examples ready to test. This provides just one example from the article. It can be done in multiple different ways. I've left the idea below, but we can go with other approaches as well.

Reply:
Modified the code with all examples for the readers to test